environments/openstack/playbook-bootstrap-images.yml

```diff
@@ -6,7 +6,7 @@
   connection: local
 
   vars:
-    url_ubuntu_image: https://cloud-images.ubuntu.com/minimal/releases/jammy/release/ubuntu-22.04-minimal-cloudimg-amd64.img
+    url_ubuntu_image: https://cloud-images.ubuntu.com/minimal/releases/noble/release/ubuntu-24.04-minimal-cloudimg-amd64.img
```

Comment:
I think it makes sense to use the ansible_ facts here to always upload the image that matches the distribution. Can you prepare a dedicated PR for this?